jrqs-app/src/java/ua/dp/isd/jrqs/SqlBuilder.java

```diff
@@ -30,25 +30,23 @@
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import ua.dp.isd.jrqs.ESearchBy.SearchBy;
 
 public class SqlBuilder {
 
-    public enum SelectBy {
-        ROWID,
-        FRID,
-        TMSTask,
-        Text
-    }
-
-    private final int MAX_RANGE_SIZE = 5000;
+    private static final int MAX_PARAMETERS = 100;
+    private static final int MAX_SEARCH_RESULTS = 100;
     private static final String SQL_SELECT_TEMPLATE
-            = "select id, fr_id, fr_tms_task, fr_object, fr_text, ccp, created, modified, status, boundary, source from requirements %s limit 100;";
+            = "select id, fr_id, fr_tms_task, fr_object, fr_text, ccp, created, modified, "
+            + "status, boundary, source from requirements %s limit "
```

Comment:
Pro tip: you can have a static sealed class with SQL table column names as const fields.